src/App.js

```diff
@@ -1,10 +1,11 @@
 import './App.css';
-import Calculator from './components/Calculator';
+// import Calculator from './components/Calculator';
+import MyRoutes from './components/MyRoutes';
 
 function App() {
   return (
     <>
-      <Calculator />
+      <MyRoutes />
```

Comment:
- Your App is working fine 👍 
![Capture](https://user-images.githubusercontent.com/63972670/221849599-2e1fdbc1-7a74-4955-9904-231c249adbb1.PNG)